src/js/handleGame.js

```diff
@@ -0,0 +1,164 @@
+import createGameBoards from "./boardView.js";
+import {UltimateBoard} from "./Boards/UltimateBoard.js";
+import {HumenPlayer} from "./Players/HumenPlayer.js";
+import {ComputerPlayer} from "./Players/ComputerPlayer.js";
+import { Board } from "./Boards/Board.js";
+
+const   boardSize = 3,
+        elementsID = {gameStatus: 'gameStatus',
+                    humPlayersign: 'humPlayerSign',
+                    compPlayersign: 'compPlayerSign'};
+let humPlayer,
+    compPlayer,
+    currPlayer,
+    bigBoard,
+    scoreBrd;
+
+
+export function startGame(mainBoardElem, scoreElem, gameInfoElem){
+    initGameBoards(mainBoardElem, scoreElem);
+    initPlayers();
+    initGameInfo(gameInfoElem);
+    updateCurrPlayer();
+}
+
+function initGameBoards(mainBoardElem, scoreElem){
+    // Creating all game boards
+    const brds = createGameBoards(boardSize);
+
+    mainBoardElem.appendChild(brds.mainBrd.tblElem);
+    scoreElem.appendChild(brds.scoreBrd.tblElem);
+
+    bigBoard = new UltimateBoard("brd-0", boardSize, brds.mainBrd.tblElem, brds.mainBrd.cellsArr);
+    scoreBrd = new Board("scrBrd", boardSize, brds.scoreBrd.tblElem, brds.scoreBrd.cellsArr);
```

Comment:
OOP principal for normalized inheritance trees (taken from _Effective Java_ book I once read): classes should either be abstract or not derived from. Following this principal, you should have a class for score board derived from "Board" class and not use it directly. Such a class can act as observer to "win" events raised by sub boards in order for the win to be reflected in the score board